common/services/prismic/exhibitions.js

```diff
@@ -417,15 +422,18 @@ export async function getExhibitionRelatedContent(
     exhibitionFields,
     articlesFields,
   ];
-  const types = ['exhibitions', 'events', 'articles', 'books'];
+  const types = ['exhibitions', 'events', 'articles', 'books', 'pages'];
   const extraContent = await getTypeByIds(req, types, ids, { fetchLinks });
   const parsedContent = parseMultiContent(extraContent.results).filter(doc => {
     return !(doc.type === 'events' && doc.isPast);
   });
 
   return {
     exhibitionOfs: parsedContent.filter(
-      doc => doc.type === 'exhibitions' || doc.type === 'events'
+      doc =>
+        doc.type === 'exhibitions' ||
+        doc.type === 'events' ||
+        doc.type === 'pages'
```

Comment:
I can't really tell why this is `exhibitionsOf`?

I think we're just looking to wedge this in somewhere, but makes me think we should think a bit about what we're modelling.

Is this for the new set of content being produced, that will be related to the exhibition?

Something like this?


![separation_of_concerns-4](https://user-images.githubusercontent.com/31692/117665636-f3e6c800-b19a-11eb-96bc-156542fbb27f.png)